### ISSUE TITLE
Cross-Site-Cookie

### ISSUE THREAD

**Issue by marcOSS040** [kiprotect/klaro#290]
With the latest Firefox (v80.0) I get an error in the console, that the Cookie soon will be treated as a Cross-Site-Cookie, because some scheme does not match.

"Das Cookie “klaro” wird bald als Cross-Site-Cookie gegen “https://xxxx/assets/js/klaro/config.js” behandelt, da das Schema nicht übereinstimmt."

**Comment by adewes** [comment]
Hi @marcOSS040! Are you on the latest version of Klaro? Do you get the same error on our website (https://kiprotect.com/klaro) when resetting the consent and clicking on "Accept" afterwards? In general we use the `Lax` same-site cookie setting (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite), I don't see any errors due to that with Firefox 80. Do you maybe use a different `cookieDomain` setting? If possible please send me a link to your site (via e-mail to klaro@kiprotect.com if you don't want to post it here) and I'll have a look at it.

**Comment by marcOSS040** [comment]
Hi @adewes, 

thanks for your prompt reply. 

I do not get any errors/warnings at your website.

Meanwhile I did some changes in the cookie presets for gAnalytics, which I configured as opt-in which also resolved the warnings for klaro. I added some cookieFlags there:

`<script type="opt-in" data-type="application/javascript" data-name="googleTagManager">
					window.dataLayer = window.dataLayer || [];
					function gtag(){dataLayer.push(arguments);}
					gtag("js", new Date());
					gtag("config", "UA-1234567-8", { "anonymize_ip": true, "cookieFlags": "secure;samesite=none" });

                </script>`

I also configured "ini_set('session.cookie_secure', 'On');" at my website.

So this is not a bug in klaro at all! Thanks for your work!

Kind regards, Marco
